02-peer-protocol.md

```diff
@@ -604,7 +604,8 @@ A sending node:
     - MUST NOT send a `shutdown`.
   - MUST NOT send multiple `shutdown` messages.
   - MUST NOT send an `update_add_htlc` after a `shutdown`.
-  - if no HTLCs remain in either commitment transaction:
+  - if neither side has a commitment for which a revocation is owed and neither
+  commitment contains any HTLCs (including dust HTLCs):
     - MUST NOT send any `update` message after a `shutdown`.
```

Comment:
Let me know what you think about this proposal, I personally find the intent clearer this way (I find the requirement below your change unclear, since it's somewhat unrelated to when we sent `shutdown`):

```suggestion
  - if no HTLCs remain in either commitment transaction (including dust HTLCs)
    and neither side has a pending `revoke_and_ack` to send:
    - MUST NOT send any `update` message after that point.
```

Reply:
I agree

Reply:
done